content/blogs/release-0-23.md

```diff
@@ -0,0 +1,711 @@
+---
+title: Kestra 0.23 introduces Multi-Panel Editor, No-Code Redesign, and Unit Testing
+description: Kestra 0.23 delivers a multi-panel editor, revamped no-code flow editor, YAML unit testing, new filters, and new plugins for a more productive orchestration experience.
+date: 2025-06-17T17:00:00
+category: News & Product Updates
+author:
+  name: Benoit Pimpaud
+  image: bpimpaud
+image: /blogs/release-0-23.jpg
+---
+
+
+The table below highlights the key features of this release.
+
+| Feature                                   | Description                                                                | Edition |
+|-------------------------------------------|----------------------------------------------------------------------------| --- |
+| Multi Panel Editor       | Enhanced visual editor with split-screen capabilities for simultaneous code and visual editing | All Edition |
+| No-Code Flow Editor | Create Kestra flows from the new form-based UI without writing code | All Edition |
+| Unit Test Flows       | Built-in testing framework for validating workflow behavior | Enterprise Edition |
+| New Filters         | New UI Filters with faster autocompletion and editable as plain text | All Edition |
+| Outputs in Internal Storage      | Store and manage workflow outputs in isolated tenant storage for complete data separation | Enterprise Edition |
+| Improved dashboard management         | Configure your own default dashboard with new customizable KPI charts and adjustable chart widths | All Edition |
+| Python Dependency Caching    | Speed up your workflows with automatic caching of script dependencies across executions - just define your dependencies and Kestra handles the rest | All Edition |
+| Manage Apps & Dashboard with Git | Version control your dashboards and apps with Git tasks | All Edition |
+
+Check the video below for a quick overview of all enhancements.
+
+<div class="video-container">
+    <iframe src="[YOUTUBE_VIDEO_URL]" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+</div>
+
+---
+
+Let's dive into these highlights and other enhancements in more detail.
+
+## Feature Highlights
```

Comment:
```suggestion
```

We don't need that H2, everything covered in this blog is a feature highlight

I'd turn all downstream from here as H2 instead to have that included in TOC